include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -164,8 +164,9 @@ def TT_LoadOp : TT_Op<"load",
         OpBuilder<(ins "Value":$ptr, "Value":$mask, "Value":$other, "triton::CacheModifier":$cache,
                        "triton::EvictionPolicy":$evict, "bool":$isVolatile)>,
         // A utility function to build the operation with all attributes
-        OpBuilder<(ins "Value":$ptr, "Value":$mask, "Value":$other, "Optional<ArrayRef<int32_t>>":$boundaryCheck,
-                       "Optional<triton::PaddingOption>":$padding, "triton::CacheModifier":$cache,
+        OpBuilder<(ins "Value":$ptr, "Value":$mask, "Value":$other,
+                       "std::optional<ArrayRef<int32_t>>":$boundaryCheck,
+                       "std::optional<triton::PaddingOption>":$padding, "triton::CacheModifier":$cache,
```

Comment:
Should we replace all `Optional` with `std::optional`?
I can also find `Optional` here: https://github.com/openai/triton/blob/9d5354d9914c5a34e05435e2cc13f88148ecc103/include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td#L223

Reply:
Yes, we should replace all C++ `llvm::Optional` with `std::optional`.
The one you linked is a TableGen `Optional` though and that should stay.